src/mars.c

```diff
@@ -1066,6 +1069,8 @@ Language changes listed by -transition=id:\n\
         global.params.debuglibname = global.params.defaultlibname;
 
 #if TARGET_OSX
```

Comment:
The Objective-C support is enabled if the makefile variable `D_OBJC` is enabled.